spec/presenters/two_factor_auth_code/generic_delivery_presenter_spec.rb

```diff
@@ -30,4 +30,22 @@ def presenter_with(arguments = {}, view = ActionController::Base.new.view_contex
       end
     end
   end
+
+  describe '#piv_cac_option' do
+    context 'for a user without a piv/cac enabled' do
+      let(:presenter) { presenter_with(has_piv_cac_configured: false) }
+
+      it 'returns nothing' do
+        expect(presenter.send(:piv_cac_option)).to be_nil
+      end
+    end
+
+    context 'for a user with a piv/cac enabled' do
+      let(:presenter) { presenter_with(has_piv_cac_configured: true) }
+
+      it 'returns something' do
+        expect(presenter.send(:piv_cac_option)).to_not be_nil
```

Comment:
If we aren't testing this elsewhere, it would be good to verify that the correct thing is returned.